From 3b158a143fc4eb7e4f736430296f8b63fd4396f5 Mon Sep 17 00:00:00 2001
From: Arun Kumar <arunkumar.ev@ltts.com>
Date: Tue, 7 Jan 2020 11:43:43 +0530
Subject: [PATCH 1/5] IMX8MQ: Modify Board configuration file

Modify board configuration file to include adlink device
tree changes.

Signed-off-by: Arun Kumar <arunkumar.ev@ltts.com>
---
 common/build/dtbo.mk         |  3 ++-
 imx8m/evk_8mq/BoardConfig.mk | 18 ++++++++++--------
 2 files changed, 12 insertions(+), 9 deletions(-)

diff --git a/common/build/dtbo.mk b/common/build/dtbo.mk
index cb2c5e1..3ccb5c0 100644
--- a/common/build/dtbo.mk
+++ b/common/build/dtbo.mk
@@ -26,7 +26,8 @@ DTS_ADDITIONAL_PATH :=
 else ifeq ($(TARGET_KERNEL_ARCH), arm64)
 KERNEL_CROSS_COMPILE := $(KERNEL_TOOLCHAIN_ABS)/aarch64-linux-androidkernel-
 KERNEL_SRC_ARCH := arm64
-DTS_ADDITIONAL_PATH := freescale
+#DTS_ADDITIONAL_PATH := freescale
+DTS_ADDITIONAL_PATH := adlink
 else
 $(error kernel arch not supported at present)
 endif
diff --git a/imx8m/evk_8mq/BoardConfig.mk b/imx8m/evk_8mq/BoardConfig.mk
index 164ebd1..5c7a90b 100644
--- a/imx8m/evk_8mq/BoardConfig.mk
+++ b/imx8m/evk_8mq/BoardConfig.mk
@@ -150,22 +150,24 @@ endif
 endif
 
 BOARD_PREBUILT_DTBOIMAGE := out/target/product/evk_8mq/dtbo-imx8mq.img
+
+TARGET_BOARD_DTS_CONFIG ?= imx8mq:adlink-lec-imx8m.dtb
 # imx8mq with HDMI display
-TARGET_BOARD_DTS_CONFIG ?= imx8mq:fsl-imx8mq-evk.dtb
+#TARGET_BOARD_DTS_CONFIG ?= imx8mq:fsl-imx8mq-evk.dtb
 # imx8mq with MIPI-HDMI display
-TARGET_BOARD_DTS_CONFIG += imx8mq-mipi:fsl-imx8mq-evk-lcdif-adv7535.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-mipi:fsl-imx8mq-evk-lcdif-adv7535.dtb
 # imx8mq with HDMI and MIPI-HDMI display
-TARGET_BOARD_DTS_CONFIG += imx8mq-dual:fsl-imx8mq-evk-dual-display.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-dual:fsl-imx8mq-evk-dual-display.dtb
 # imx8mq with MIPI panel display
-TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-panel:fsl-imx8mq-evk-dcss-rm67191.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-panel:fsl-imx8mq-evk-dcss-rm67191.dtb
 # imx8mq rev. B3 board with HDMI display
-TARGET_BOARD_DTS_CONFIG += imx8mq-b3:fsl-imx8mq-evk-b3.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-b3:fsl-imx8mq-evk-b3.dtb
 # imx8mq rev. B3 board with mipi-hdmi display
-TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-b3:fsl-imx8mq-evk-lcdif-adv7535-b3.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-b3:fsl-imx8mq-evk-lcdif-adv7535-b3.dtb
 # imx8mq rev. B3 board with MIPI panel display
-TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-panel-b3:fsl-imx8mq-evk-dcss-rm67191-b3.dtb
+#TARGET_BOARD_DTS_CONFIG += imx8mq-mipi-panel-b3:fsl-imx8mq-evk-dcss-rm67191-b3.dtb
 # u-boot target for imx8mq_evk
-TARGET_BOOTLOADER_CONFIG := imx8mq:imx8mq_evk_android_defconfig
+#TARGET_BOOTLOADER_CONFIG := imx8mq:imx8mq_evk_android_defconfig
 
 TARGET_KERNEL_DEFCONFIG := android_defconfig
 # TARGET_KERNEL_ADDITION_DEFCONF ?= android_addition_defconfig
-- 
2.7.4

